tests/gold_tests/pluginTest/header_rewrite/gold/set_body_status.gold

```diff
@@ -0,0 +1,16 @@
+``
+> GET ``
+> Host: www.example.com``
+> User-Agent: curl/``
+> Accept: */*
+> Proxy-Connection: Keep-Alive
+``
+< HTTP/1.1 200 OK
+< Date: ``
+< Proxy-Connection: keep-alive
+< Server: ATS/``
+< Cache-Control: no-store
+< Content-Type: text/html
+< Content-Language: en
+< Content-Length: 3
+``
```

Comment:
I think we're also expecting a body here with the string "200".

Reply:
Ok. I was going to add that but it needed an extra file in the way things worked, and I figure the length would be sufficient :) I'll take a look again to see if there's an easy way to capture the stdout in this new infrastructure.